canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -517,6 +517,7 @@ class PropertiesMain extends React.Component {
 				customPanels={this.props.customPanels}
 				rightFlyout={this.props.propertiesConfig.rightFlyout}
 				categoryView={this.props.propertiesConfig.categoryView}
+				showAlertsTab={(typeof this.props.propertiesConfig.showAlertsTab !== "undefined") ? this.props.propertiesConfig.showAlertsTab : true}
```

Comment:
Instead of doing this check couldn't we do 
```
showAlertsTab={ this.props.propertiesConfig.showAlertsTab !== false }
```
The in all cases besides `false` this would be true.